### DISCUSSION TITLE
Date format issue

### DISCUSSION THREAD

**Discussion by prykon** [DiscipleTools/disciple-tools-meetings#2] [Q&A] [answered]
What's the format for adding a date in textbox? I tried with YYYYMMDD but it throws the following error.

`{"code":"create_post","message":"'notes' field expected to be an array","data":null}`

We should makes this more intuitive (using a calendar module, perhaps?).

**Comment by prykon** [comment]
Now it's showing the calendar module. Weird.

**Comment by corsacca** [comment, marked as answer]
should be fixed by: https://github.com/DiscipleTools/disciple-tools-meetings/commit/b76a589ff249736049d7d2042e07b99d8681f277
